.github/workflows/sync-sendgripd-template.yaml

```diff
@@ -0,0 +1,38 @@
+name: Sync handlebars template with SendGrid
+on:
+  workflow_dispatch:
+    inputs:
+      forceSyncAll:
+        description: Force sync all files in templates dir (Useful for initial synchronization)
+        required: false
+        default: false
+jobs:
+  sendgrid_sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: SendGrid sync
+        id: sendgrid_sync
+        uses: nanopx/action-sendgrid-sync@0.5.2
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          sendgridApiKey: ${{ secrets.SENDGRID_API_KEY }}
+          templatesDir: 'emailTemplates/'
+          forceSyncAll: '${{ github.event.inputs.forceSyncAll }}'
```

Comment:
What is forceSyncAll? When would we want to provide it? Could it update templates which are used by other teams?

Reply:
We manually set it to true when we add a new template, otherwise this sync job only updates the template with existing name matched in sendgrid

Reply:
Maybe add something to the readme about when to send a forceSyncAll and how to do it.